ansible/setup.yml

```diff
@@ -81,9 +81,21 @@
         update_cache: yes
       with_items:
         - build-essential
+        - python-yaml
       tags:
         - dev-tools
 
+    - name: "Create custom fact directory"
+      file:
+        path: "/etc/ansible/facts.d"
+        state: "directory"
+
+    - name: "Install epoch fact file"
```

Comment:
I see you do not include
```
- name: "Re-run setup to use custom facts"
  setup: ~
```
(from https://medium.com/@jezhalford/ansible-custom-facts-1e1d1bf65db8 linked in https://www.pivotaltracker.com/n/projects/2152594/stories/156535422 )

as these facts are are meant to be used by subsequent PR amending playbook in https://github.com/aeternity/epoch , right?

Reply:
Yes, we don't need the new facts in this playbook.